cbmc/Makefile.common

```diff
@@ -949,6 +949,7 @@ result:
 	$(MAKE) -B _result
 	@ echo Running 'litani build'
 	$(LITANI) run-build
+	tail -4 $(LOGDIR)/result.txt
```

Comment:
What's this for?

Reply:
On a successful proof run, it shows
```

** 0 of 394 failed (1 iterations)
VERIFICATION SUCCESSFUL

```
which is what I want to see. Without that line I find myself having to type
```
less logs/result.txt
```
after every single `make result` run, so I decided to make it do that every time.